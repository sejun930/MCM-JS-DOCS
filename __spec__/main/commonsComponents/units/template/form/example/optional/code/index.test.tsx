import { render } from "@testing-library/react";
// import { RecoilRoot } from "recoil";
import _ExampleOptionalCodeIconPage from "src/main/commonsComponents/units/template/form/example/optional/code";

export default describe("Example Code Icon", () => {
  // 스냅샷
  test("Example Code Icon - Snapshot", () => {
    const { container } = render(
      <_ExampleOptionalCodeIconPage
        showCode={false}
        toggleShowCode={() => {}}
      />
    );
    expect(container).toMatchSnapshot();
  });

  // 스냅샷
  test("Example Code Icon - Snapshot", () => {
    const { container } = render(
      <_ExampleOptionalCodeIconPage
        showCode={false}
        toggleShowCode={() => {}}
      />
    );
    expect(container).toMatchSnapshot();
  });
});
