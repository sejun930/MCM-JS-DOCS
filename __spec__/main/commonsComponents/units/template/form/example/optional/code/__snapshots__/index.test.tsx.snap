// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example Code Icon Example Code Icon - Snapshot 1`] = `
<div>
  <button
    class="_button_ _example_code_icon_btn_"
    role="button_click_event"
  >
    <div
      class="css-16pyd80"
    >
      <span
        class="css-1iywk7f"
      >
        &lt;
      </span>
      <span
        class="css-9alhxk"
      >
        /
      </span>
      <span
        class="css-1t3qs2b"
      >
        &gt;
      </span>
    </div>
  </button>
</div>
`;
